packages/cli/src/cli-default-servient.ts

```diff
@@ -98,10 +98,7 @@ export default class DefaultServient extends Servient {
 
         // apply flags
         if (clientOnly) {
-            if (!this.config.servient) {
-                this.config.servient = {};
-            }
-            this.config.servient.clientOnly = true;
+            this.config.servient ??= { clientOnly };
```

Comment:
are you sure this works as the previous code? In my understanding previously if `this.config.servient` was already defined we just set `clientOnly` property to `true`. With the new code if it is defined we do not override it.

Reply:
Oh, you are right, then we guess we need to keep something like this here:

```suggestion
            this.config.servient ??= {};
            this.config.servient.clientOnly = true;
            
```